package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "dotnet-build",
-    "version": "5.0.0",
+    "version": "5.1.0",
```

Comment:
This is by no means a minor upgrade. 
This breaks any of our projects that use tests, none of the tests will be ran and the results will not be available in bitbucket even if they do somehow run.

Reply:
Adding a unittest harness should not be breaking, we can use an alpha or something for compatabilty.

Reply:
Depends on the definition of break I guess.
There are two things that would be broken if this was merged:

- Tests that don't fit the new filter (integration category) would no longer be run on commit. (though I thought this was much worse but it's a != instead of ==)
- Bitbucket integration on the test/coverage result (because it won't find the test-results in the correct (pre-defined) folder)

This would apply to all our other software as soon as they upgrade the container.

Reply:
Good thought! To be tested for sure:-)